src/gui/owncloudsetupwizard.cpp

```diff
@@ -116,7 +116,7 @@ void OwncloudSetupWizard::startWizard()
     // if its a relative path, prepend with users home dir, otherwise use as absolute path
 
     if (!QDir(localFolder).isAbsolute()) {
-        localFolder = QDir::homePath() + QDir::separator() + localFolder;
+        localFolder = QDir::homePath() + "/" + localFolder;
```

Comment:
This was actually correct, why change it? Also this leads to more expensive concatenations.

Reply:
Without this change I get on Windows for example the following path `C:/Users/felix\Nextcloud`. Which bites with my logic to remove the home path. https://doc.qt.io/qt-5/qdir.html#separator

Reply:
Yes indeed... my bad. Please go for `QLatin1Char('/')` though.